modules/installation-restricted-network-samples.adoc

```diff
@@ -29,6 +29,11 @@ part of the install payload, are not managed by the Cluster Samples Operator. Th
 not addressed in this procedure.
 ====
 
+[IMPORTANT]
+====
+The Cluster Samples Operator must be set to `Managed` in a disconnected environment. To install the image streams, you have a mirrored registry.
```

Comment:
**question:** @Amrita42 is this `mirrored` registry or mirror registry. 

Based on the links from Bugzilla I'm wondering if this is correct. I'll be happy to merge if this is the correct usage here.

Reply:
We have usage of mirrored registry for example >> https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html

Reply:
please go ahead with the merge. thanks